cmd/controller.go

```diff
@@ -69,6 +74,20 @@ func parseControllerConfig() {
 	if mountPodImage := os.Getenv("JUICEFS_MOUNT_IMAGE"); mountPodImage != "" {
 		config.MountImage = mountPodImage
 	}
+
+	k8sclient, err := k8s.NewClient()
+	if err != nil {
+		klog.V(5).Infof("Can't get k8s client: %v", err)
+		os.Exit(0)
+	}
+	ds, err := k8sclient.AppsV1().DaemonSets(config.Namespace).Get(context.TODO(), JUICEFS_CSI_NODE_NAME, metav1.GetOptions{})
```

Comment:
Can you move Kubernetes client related function in [`pkg/k8sclient/client.go`](https://github.com/juicedata/juicefs-csi-driver/blob/master/pkg/k8sclient/client.go) to keep the structure of the project intact ?

Reply:
Ok, updated.